main.py

```diff
@@ -192,7 +186,7 @@ def main():
         "latest version.", dest="update", nargs="?", const="", required=False)
 
     parser.add_argument('-bu', '--backup', help="Backup your Minecraft Server", 
-        dest="path", nargs="?", const="Default", type=str, required=False)
+        dest="path", action='store_true', required=False)
```

Comment:
So you think giving the user, which I realize is use at present, the ability to choose a location to do a manual backup offers no value? Or do you think it would be better down the road?

Reply:
Given that they can change where the backups are stored via the config.yml, I don't really see a benefit in it, no.